extensions/amp-auto-ads/0.1/amp-auto-ads.js

```diff
@@ -44,6 +47,8 @@ export class AmpAutoAds extends AMP.BaseElement {
       return;
     }
 
+    extensionsFor(this.win).loadElementClass(AD_TAG);
```

Comment:
discussed with @dvoytenko offline, we think 
1) performance impact is minimum given that amp-ad is blocked by amp-auto-ads config loading in anyway
2) auto insert amp-ad gives us chance to load amp-ad even smarter in future (we are seeking for split amp-ad extension for different ad networks).